enterprise-suite/Makefile

```diff
@@ -43,7 +43,7 @@ script-tests: ## Run installer tests
 
 .PHONY: frontend-tests
 frontend-tests: ## Run the frontend tests against minikube
```

Comment:
The comment here and in the next two targets should be modified.  (e.g. first one runs all tests, second one runs first subset, third one runs second subset.)

Reply:
I'd still like to see this change but no biggy